### PR TITLE
Implement swipe match backend

### PR DESCRIPTION
**Pull request by alexcostaluiz**
draft

### CODE REVIEW

File: project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        for (AsyncContext asyncContext : asyncContexts.values()) {
+          try {
+            sendUser(asyncContext.getResponse().getWriter(), user);
+          } catch (Exception e) {
+            asyncContexts.values().remove(asyncContext);
+          }
+        }
+      } catch (InterruptedException e) {
+        // idk yet
```
Comment (kpancha) [inline project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java:47]: What is an example of when this block would execute (to help us figure out how to handle the exception)?

File: project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    }
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String username = request.getParameter("username");
+    try {
+      queue.put(username);
+    } catch (InterruptedException e) {
+      // failed, idk yet
```
Comment (kpancha) [inline project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java:114]: We could probably log this with Logger, at least for now

File: project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          }
+        }
+      } catch (InterruptedException e) {
+        // idk yet
+      }
+    }
+  });
+
+  @Override
+  public void destroy() {
```
Comment (miahsanchez) [inline project/backend/src/main/java/com/google/sps/servlets/SwipeMatchServlet.java:53]: when would we be calling this?

**Comment by alexcostaluiz** [comment]
This method of implementing swipe match did not work with our appengine setup.

**Pull request closed without merging**